<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>miscEditor</class>
 <widget class="QDialog" name="miscEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Misc Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget_editor">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_main">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>lineEdit_name</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="SmartLineEdit" name="lineEdit_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>100</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_type">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>comboBox_type</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="BtComboBox" name="comboBox_type">
         <property name="toolTip">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_amountIsWeight">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Check it if the amount listed is in kg instead of L.</string>
         </property>
         <property name="text">
          <string>Amount is weight?</string>
         </property>
         <property name="buddy">
          <cstring>checkBox_amountIsWeight</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QCheckBox" name="checkBox_amountIsWeight">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Checked if the given amount is weight instead of volume</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>

       <item row="4" column="2">
        <widget class="SmartLabel" name="label_inventory">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Amount in Inventory</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_inventory</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="SmartLineEdit" name="lineEdit_inventory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Amount in inventory</string>
         </property>
        </widget>
       </item>

       <item row="4" column="5">
        <widget class="QLabel" name="label_amountType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Whether the amount is weight or volume</string>
         </property>
         <property name="text">
          <string>Amount Type</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_amountType</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="6">
        <widget class="BtComboBox" name="comboBox_amountType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Whether the amount is weight or volume</string>
         </property>
        </widget>
       </item>

       <item row="5" column="0">
        <widget class="SmartLabel" name="label_time">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>lineEdit_time</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="SmartLineEdit" name="lineEdit_time">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>100</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Time</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_producer">
         <property name="text">
          <string>Producer</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_producer</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="SmartLineEdit" name="lineEdit_producer">
         <property name="toolTip">
          <string>Producer</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="label_productId">
         <property name="text">
          <string>Product ID</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_productId</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="SmartLineEdit" name="lineEdit_productId">
         <property name="toolTip">
          <string>Product ID</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_usefor">
      <attribute name="title">
       <string>Use for</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit_useFor"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_notes">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="textEdit_notes"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_new">
       <property name="toolTip">
        <string>New misc</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/smallPlus.svg</normaloff>:/images/smallPlus.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="toolTip">
        <string>Save and close</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/filesave.svg</normaloff>:/images/filesave.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="toolTip">
        <string>Discard and close</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/exit.svg</normaloff>:/images/exit.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/BtComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>SmartLabel</class>
   <extends>QLabel</extends>
   <header>widgets/SmartLabel.h</header>
   <slots>
    <signal>changedSystemOfMeasurementOrScale(PreviousScaleInfo)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>SmartLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/SmartLineEdit.h</header>
   <slots>
    <slot>lineChanged(PreviousScaleInfo)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget_editor</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>comboBox_type</tabstop>
  <tabstop>lineEdit_time</tabstop>
  <tabstop>lineEdit_inventory</tabstop>
  <tabstop>pushButton_new</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>textEdit_useFor</tabstop>
  <tabstop>textEdit_notes</tabstop>
 </tabstops>
 <resources>
  <include location="../brewken.qrc"/>
 </resources>
</ui>
